EDDNResponder/EDDNResponder.cs

```diff
@@ -106,32 +154,28 @@ private void handleRawEvent(Event theEvent)
             IDictionary<string, object> data = Deserializtion.DeserializeData(theEvent.raw);
             string edType = JsonParsing.getString(data, "event");
 
-            if (edType == "FSDTarget" || edType == "StartJump")
-            {
-                // FSDTarget events describing the system we are targetting rather than the system we are in.
-                // Scan events can register after StartJump and before we actually leave the originating system.
-                // These must be ignored.
-                return;
-            }
-            else if (edType == "Location" || edType == "FSDJump")
+            // Ignore any events that we've blacklisted for contaminating our location data
+            if (ignoredEvents.Contains(edType)) { return; }
```

Comment:
A HashSet would offer O(1)  lookup instead of O(N) for an array, but for these sizes of arrays it is a bit moot.